task13/index.js

```diff
@@ -0,0 +1,96 @@
+const parent = document.getElementById("js-parent"); 
+
+const appendModalForParent = () => {
+    const modalPlace = document.createElement('section');
+    const modalText = document.createElement('p');
+    const closeBtn = document.createElement('input');
+    modalPlace.style.margin = '0 auto';
+    modalPlace.style.textAlign = "center";
+    modalPlace.style.width = '300px';
+    modalPlace.style.padding = '20px';
+    modalPlace.style.backgroundColor = "lightGray";
+    modalPlace.id = "modalPlace";
```

Comment:
IMO
why this modal create here, you can create modal with html and css In advance and display: none, you can just add dispay block here.

Reply:
I fixed it using HTML and CSS.
I tried to turn off the display with display: none, but I don't know if this process is correct.